src/applications/ivc-champva/10-7959C/tests/config/form.unit.spec.jsx

```diff
@@ -340,3 +340,12 @@ describe('FileFieldWrapped', () => {
     expect(ffw).to.not.be.undefined;
   });
 });
+
+testNumberOfWebComponentFields(
+  formConfig,
+  formConfig.chapters.formSignature.pages.formSignature.schema,
+  formConfig.chapters.formSignature.pages.formSignature.uiSchema,
+  1,
+  'Form signature page',
+  { ...mockData.data },
+);
```

Comment:
Unrelated, but added unit test verifying the "who's signing this document" page adds the right number of web components to the page.